Applications/Scale/test/testScale.cpp

```diff
@@ -460,4 +470,328 @@ void scalePhysicalOffsetFrames()
+
+        // Scale the model.
+        model->scale(s, scaleSet, false);
+        expectedMarkerLoc =
+            expectedMarkerLoc.elementwiseMultiply(Vec3(scaleFactor));
+
+        // Ensure all Markers are coincident after scaling.
+        testMarkerLoc(marker1, s, "after scaling");
+        testMarkerLoc(marker2, s, "after scaling");
+        testMarkerLoc(marker3, s, "after scaling");
+    }
+
+    // Case 5: Attach a PathActuator to a PhysicalOffsetFrame. The two
+    //         PathActuators in this example model should be coincident before
+    //         and after scaling.
+    {
+        cout << "- case 5" << endl;
+
+        Model* model = new Model();
+        OpenSim::Body* body = new OpenSim::Body("body", 1, Vec3(0), Inertia(0));
+        model->addBody(body);
+
+        FreeJoint* free = new FreeJoint("free", model->getGround(), *body);
+        free->setName("free");
+        model->addJoint(free);
+
+        // First PathActuator is attached to and owned by the Body.
+        const Vec3 offset1 = Vec3(0.2, 0.4, 0.6);
+        PathActuator* act1 = new PathActuator();
+        act1->setName("pathActuator1");
+        act1->addNewPathPoint("point1a", model->updGround(), Vec3(0));
+        act1->addNewPathPoint("point1b", *body, offset1);
+        body->addComponent(act1);
+
+        // Second PathActuator is attached to the Body via a POF. Both new
+        // ModelComponents are owned by the first PathActuator.
+        const Vec3 offset2 = offset1 / 2.;
+        const Transform tf2 = Transform(offset2);
+
+        PhysicalOffsetFrame* pof2 = new PhysicalOffsetFrame("pof2", *body, tf2);
+        act1->addComponent(pof2);
+
+        PathActuator* act2 = new PathActuator();
+        act2->setName("pathActuator2");
+        act2->addNewPathPoint("point2a", model->updGround(), Vec3(0));
+        act2->addNewPathPoint("point2b", *pof2, offset2);
+        act1->addComponent(act2);
+
+        State& s = model->initSystem();
+
+        // Ensure PathPoints are coincident before scaling.
+        auto testPathPointLoc =
+            [&](const State& s, const std::string& msg) -> void
+        {
+            const PathPointSet& pps1 = act1->getGeometryPath().getPathPointSet();
+            const PathPointSet& pps2 = act2->getGeometryPath().getPathPointSet();
+
+            for (int i = 0; i < 2; ++i)
+            {
+                const Vec3& p1 = pps1[i].getLocationInGround(s);
+                const Vec3& p2 = pps2[i].getLocationInGround(s);
+
+                ASSERT_EQUAL(p1, p2, SimTK::SignificantReal, __FILE__, __LINE__,
+                    "The location of point " + std::to_string(i)
+                    + " does not match (" + msg + ")\n  PathActuator1: "
+                    + p1.toString() + "\n  PathActuator2: " + p2.toString());
+            }
+        };
+
+        testPathPointLoc(s, "before scaling");
+
+        // Scale the model.
+        model->scale(s, scaleSet, false);
```

Comment:
Minor point, but you might want to add a comment that indicates that `Model::scale()` updates the properties of the same model and re-inits the System and updates the the input state.  In the future, I hope that scaling just returns a new model and leaves it to the consumer of the scaled model to `initSystem()` or not, especially if it plans to perform more edits.  If/when that happens the lambda will break (which is OK) and a comment here will shed light as to why (i.e. `act1` and `act2` will still be pointing to the old model).

Reply:
I was unaware you were planning this API change. I'll add the comment here, but I think there will be a lot of (uncommented) code to change other than this lambda.

Reply:
True! I was just surprised that `act1` and `act2` were not stale on the call to `testPathPointLoc` after scaling. I am not planning to make any API changes for 4.0.

Reply:
`act1` and `act2` are defined inside the lambda. How would they be stale on repeated calls to the function?

Reply:
I see what you mean: I'll fix that!

Reply:
Scaling doesn't destroy the Model, so the pointers to the two PathActuators remain valid. Not sure I understand why `scale()` would create a new model (it seems like it should be the responsibility of the caller to create a duplicate if it wants to retain an unscaled version). Regardless, I've updated the test to :fish: the PathActuators out of the model for the sake of the future developer. Bonus marks if you caught the two puns :wink: